src/components/vm/disks/diskEdit.tsx

```diff
@@ -206,7 +206,6 @@ export const EditDiskAction = ({
     }
 
     const enabled = (Object.keys(diskBusTypes).includes(disk.device || "") &&
-                     supportedDiskBusTypes &&
```

Comment:
that's because the type system  guarantees that it's always non-null?

Reply:
No, because I changed libvirt-xml-parse to always return a `string[]` for this, to simplify things. TypeScript unfortunately doesn't guarantee anything at run-time.